configs/SPRACINGH7EXTREME/config.h

```diff
@@ -19,9 +19,11 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+#pragma once
+
 #define FC_TARGET_MCU     STM32H750
 
-#define BOARD_NAME        SPRACINGH7EXTREME
+#define BOARD_NAME        "SPRACINGH7EXTREME"
```

Comment:
This change won't be needed anymore.

```suggestion
#define BOARD_NAME        SPRACINGH7EXTREME
```